src/DotLiquid.Tests/StandardFilterTests.cs

```diff
@@ -414,6 +416,44 @@ public void TestSort_OnDictionaryWithPropertyOnlyInSomeElement_ReturnsSortedDict
             Assert.AreEqual(hash2["content"], result[2]["content"]);
         }
 
+        [Test]
+        public void TestSort_Indexable()
+        {
+            var packages = new [] {
+                new Package(numberOfPiecesPerPackage: 2, test: "p1"),
+                new Package(numberOfPiecesPerPackage: 1, test: "p2"),
+                new Package(numberOfPiecesPerPackage: 3, test: "p3"),
+            };
+            var expectedPackages= packages.OrderBy(p => p["numberOfPiecesPerPackage"]).ToArray();

```

Comment:
nit: avoid uncommon acronyms or single letter variables and use full English words instead for code readability. Here 'package' instead of 'p'.